marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/CreateGranularPrivilegesCommand.java

```diff
@@ -95,9 +95,10 @@ public Integer getExecuteSortOrder() {
         return Integer.MAX_VALUE;
     }
 
+    // Granular privileges should be removed before dbs as the 'granularPrivileges' map's key uses db id
     @Override
     public Integer getUndoSortOrder() {
-        return SortOrderConstants.DELETE_PRIVILEGES;
+        return SortOrderConstants.DELETE_OTHER_DATABASES - 1;
```

Comment:
If needed in the future, we could have this executed right at the beginning of undeployment too.